narayana-release-process.sh

```diff
@@ -126,9 +110,13 @@ then
     exit
   fi
   set -e
-  echo "Checking if there were any failed jobs, this may be interactive so please stand by"
-  JIRA_HOST=issues.redhat.com JENKINS_JOBS=narayana,narayana-catelyn,narayana-documentation,narayana-hqstore,narayana-jdbcobjectstore,narayana-quickstarts,narayana-quickstarts-catelyn\
-      ./scripts/release/pre_release.py
+  
+  read -p "Please check for Jira blockers (https://issues.redhat.com/issues/?jql=project%20%3D%20JBTM%20AND%20priority%20%3D%20Blocker%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC) and failed CI jobs before continuing. Continue? y/n " NOBLOCKERS
```

Comment:
@marcosgopen would you be open to narrowing this query to check for 7.next fix version rather than just all open blockers? If so we should make sure it's also documented that any issue expected to block the next release has Blocker priority with 7.next fix version even if the fix is not provided yet. That should probably go in CONTRIBUTING.md (/cc @jmfinelli given https://issues.redhat.com/browse/JBTM-3888?focusedId=25043629&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25043629)

Reply:
Thanks Tom for your reviews! I updated the script, but I am not sure about setting the Fix Version for a blocker issue before having it resolved. That might increase the complexity for developers when creating a Jira issue and might become quite an error-prone procedure. If this is the only option I am ok to approve it tough. Thanks

Reply:
Thank you for the observation. The alternative would seem to be to check the full blocker list which could be arbitrarily large? As an example of why it might be large could be lots of blockers for features that are not yet released. We would need some way to parse that out?

Reply:
Even tough the blocker list might be large I think (and hope) that the blocker list of unresolved issues would not be that large (e.g. https://issues.redhat.com/browse/JBTM-3889?jql=project%20%3D%20JBTM%20AND%20priority%20%3D%20Blocker%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC). WDYT @mmusgrov and @jmfinelli ?

Reply:
Note that the CONTRIBUTING.md document says that `The release coordinator will update this Fix Version field to correspond with the actual version being released` and that is a catch all statement that covers what I have just said in my last comment.

Reply:
But I forget why we went for this policy on the "Fix Version" field. Naively I would have asked the "fixer" to set the target release and then the release coordinator would move any issues not fixed to the next release. @tomjenkinson do you recall why we didn't go for this policy?

Reply:
But I'm not asking for a policy change since the current policy works even if it does occasionally cause confusion.

Reply:
I don't think the JBTM JIRA project has "Target Release" field yet but we can request that to be added and we could use it for identifying issues that we intend to include in the next release.

OTOH if we just would use it for blockers, I am also fine with us choosing to simulate a "Target Release" field by asking people to set "Fix Version" to "7.next" and the issue priority to Blocker. If the issue is unresolved, we know we need to wait.